### PR TITLE
ArgoCD RBAC for maneki

### PR DESCRIPTION
**Pull request by masa213f**
- Add an ArgoCD project for Maneki team.
- Permit just the Neco team to access the ArgoCD default project.

### CODE REVIEW

File: team-management/base/maneki/project.yaml [hunk 1]
```diff
@@ -0,0 +1,26 @@
+apiVersion: argoproj.io/v1alpha1
+kind: AppProject
+metadata:
+  name: maneki
+  namespace: argocd
+spec:
+  sourceRepos:
+  - '*'
+  destinations:
+  - namespace: maneki
```
Comment (zoetrope) [inline team-management/base/maneki/project.yaml]: I think `sandbox` namespace is necessary.

File: team-management/base/maneki/project.yaml [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    server: https://kubernetes.default.svc
+  namespaceResourceBlacklist:
+  - group: ''
+    kind: ResourceQuota
+  - group: ''
+    kind: LimitRange
+  - group: ''
+    kind: NetworkPolicy
```
Comment (zoetrope) [inline team-management/base/maneki/project.yaml]: The group of `NetworkPolicy` is `extensions` and `networking.k8s.io`.
Do you need to specify it?

File: team-management/base/maneki/project.yaml (hunk 1 above)
Comment (masa213f) [reply team-management/base/maneki/project.yaml]: Yes!

File: team-management/base/maneki/project.yaml (hunk 2 above)
Comment (masa213f) [reply team-management/base/maneki/project.yaml]: We need `networking.k8s.io`!

**Pull request merged**